Xamarin.Forms.Material.Tizen/Shell/MaterialNavigationView.cs

```diff
@@ -3,18 +3,65 @@
 using ElmSharp;
 using Tizen.NET.MaterialComponents;
 using Xamarin.Forms.Platform.Tizen;
+using NImage = Xamarin.Forms.Platform.Tizen.Native.Image;
 
 namespace Xamarin.Forms.Material.Tizen
 {
 	public class MaterialNavigationView : MNavigationView, INavigationView
 	{
+		ImageSource _bgImageSource;
+		NImage _bg;
+		Aspect _bgImageAspect;
+
 		IDictionary<MItem, Element> _flyoutMenu = new Dictionary<MItem, Element>();
 
 		public MaterialNavigationView(EvasObject parent) : base(parent)
 		{
 			MenuItemSelected += OnSelectedItemChanged;
 		}
 
+		public Aspect BackgroundImageAspect
+		{
+			get
+			{
+				return _bgImageAspect;
+			}
+			set
+			{
+				_bgImageAspect = value;
+				if (_bg != null)
+				{
+					_bg.ApplyAspect(_bgImageAspect);
+				}
+			}
+		}
+
+		public ImageSource BackgroundImageSource
+		{
+			get
+			{
+				return _bgImageSource;
+			}
+			set
+			{
+				_bgImageSource = value;
+				if(_bgImageSource != null)
+				{
+					if(_bg == null)
+					{
+						_bg = new NImage(this);
```

Comment:
use ElmSharp.Image instead.

Reply:
Fixed. For this change, LoadFromImageSourceAsync() method is moved into ImageExtensions from Native.Image, please review again.